docker/bootstrap/dependencies.sh

```diff
@@ -31,7 +31,8 @@ sudo -u postgres createdb -O digdag_test digdag_test
 apt-get -y install python python-pip python-dev
 pip install pip --upgrade && hash -r pip # rehashed https://github.com/pypa/pip/issues/5240
 # Using sphinx==1.4.9 because sphinx_rtd_theme with sphinx 1.5.x has a problem with search and its fix is not released: https://github.com/snide/sphinx_rtd_theme/pull/346
```

Comment:
Changes look good to me, but this comment might confuse. 
We can be deleted now because this PR is merged and released.
https://github.com/snide/sphinx_rtd_theme/pull/346

Related links:
- https://github.com/sphinx-doc/sphinx/issues/3203
- https://github.com/rtfd/sphinx_rtd_theme/blob/0.4.3/sphinx_rtd_theme/layout.html#L49

Reply:
@szyn Thanks for your pointing out.
Yes, I also noticed it, but just in case I remained it.
But I will remove it with your advice.